openstack_controller/datadog_checks/openstack_controller/openstack_controller.py

```diff
@@ -131,39 +132,44 @@ class OpenStackControllerCheck(AgentCheck):
     def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
 
+        skip_proxy = not is_affirmative(init_config.get('use_agent_proxy', True))
+        self.proxy_config = None if skip_proxy else self.proxies
```

Comment:
Proxy config should be per url https://github.com/DataDog/integrations-core/blob/64f47f3af8cd465b900a070b463f4c86cca46d09/datadog_checks_base/datadog_checks/base/checks/base.py#L143-L155

Reply:
Are you saying that we should use the `get_instance_proxy` method instead of `use_agent_proxy`?